src/features/favorites/components/__tests__/NoFavoritesResult.test.tsx

```diff
@@ -29,14 +29,14 @@ jest.mock('features/favorites/pages/FavoritesWrapper', () => ({
 describe('NoFavoritesResult component', () => {
   it('should show the message', () => {
     const text = render(<NoFavoritesResult />).getByText(
-      `Retrouve toutes tes offres en un clin d’oeil en les ajoutant à tes favoris !`
+      `Retrouve toutes tes offres en un clin d’oeil`
     )
     expect(text).toBeTruthy()
   })
 
   it('should navigate to Search when pressing button and log event', async () => {
     const renderAPI = render(<NoFavoritesResult />)
-    const button = renderAPI.getByText('Explorer les offres')
+    const button = renderAPI.getByText('Découvrir le catalogue')
     await fireEvent.press(button)
```

Comment:
Ce serait top d'update les tests et snapshots dans le même commit que le changement, ça peut aider les relecteurs, et lorsqu'on débug c'est aussi plus simple d'avoir des commits ou les tests passent 😄 (même si avec le squash and merge au final ça a moins d'importance...)

Reply:
Je fais ça parce que parfois il y a beaucoup trop de snapshots. Mais ça marche pour la prochaine fois je vais faire comme ça !